assets/js/components/KeyMetrics/ChangeMetricsLink.stories.js

```diff
@@ -43,7 +44,9 @@ export default {
 		( Story ) => {
 			const setupRegistry = ( registry ) => {
 				provideUserAuthentication( registry );
-				provideKeyMetrics( registry, { widgetSlugs: [ 'metricA' ] } );
+				provideKeyMetrics( registry, {
+					widgetSlugs: [ KM_ANALYTICS_NEW_VISITORS ],
```

Comment:
Not related to this PR, but I noticed this story needed updating to follow on latest change for filtering out non ACR metrics when feature flag is not enabled, included real non ACR metric slug to fix the story, it is a tiny change so I included it here quickly